components/script/dom/globalscope.rs

```diff
@@ -3247,7 +3247,7 @@ impl GlobalScope {
                             false,
                             can_gc,
                         );
-                        navigator.set_gamepad(selected_index as usize, &gamepad);
+                        navigator.set_gamepad(selected_index as usize, &gamepad, can_gc);
```

Comment:
This inside of a task! macro, so we don't actually want to make this change. It looks like the earlier use of can_gcis incorrect as well, so I think we want to remove the argument to handle_gamepad_connect instead.